src/interfaces/index.ts

```diff
@@ -8,16 +13,24 @@ export interface IPaginationOptions<CustomMetaType = IPaginationMeta> {
    */
   page: number | string;
   /**
-   * a babasesic route for generating links (i.e., WITHOUT query params)
+   * a basic route for generating links (i.e., WITHOUT query params)
    */
   route?: string;
 
+  /**
+   * For transforming the default meta data to a custom type
+   */
   metaTransformer?: (meta: IPaginationMeta) => CustomMetaType;
 
   /**
    * routingLabels for append in links (limit or/and page)
    */
   routingLabels?: IPaginationOptionsRoutingLabels;
+  
+  /**
+   * Used for changing query method to limit/offset (defaults to take/skip if no argument supplied)
```

Comment:
Hi @bashleigh , just found this when looking into the 2.6.0 update of this repo.
Is this comment or the default of `paginationType` correct? It says here take/skip is the default, but resolveOptions contains the following:
```ts
const paginationType = options.paginationType || PaginationTypeEnum.LIMIT_AND_OFFSET;
```

Reply:
oh yea that doesn't look right. To prevent breaking changes we left it as LIMIT by default, I'll fix the docs and tests. Thanks for pointing this out!